packages/base/components/Image/ImageWithAttribution.vue

```diff
@@ -52,8 +52,15 @@ const props = defineProps({
 </script>
 <template>
   <figure>
+    <ImageLottiePlayer
+      v-if="
```

Comment:
Given that src is a required prop, aren't the two v-if checks for it redundant?

Reply:
Good point. Checks are removed.